Doc/library/test.rst

```diff
@@ -195,7 +199,7 @@ tests.
 .. note::
 
    The :mod:`test.test_support` module has been renamed to :mod:`test.support`
-   in Python 3.x and 2.7.13.  The name ``test.test_support`` has been retained
+   in Python 3.x and 2.7.14.  The name ``test.test_support`` has been retained
```

Comment:
This change is ok: the commit a694e092f6356970ca0d3e86000a3a829fb212b4 is not part of Python 2.7.13.